server/models.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const UserSchema = mongoose.Schema(
+    {
+        username: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,  
+        },
+        password : {
+            type: String,
+            required: true,
+        }
+    }
+);
+
+const User = mongoose.model('User', UserSchema); 
+
+const EntrySchema = mongoose.Schema(
```

Comment:
Some more ideas here:
- title
- rename "entryText" to "description"
- add a `createdAt` that uses Date.now() for a cleaner timestamp

Other fun things to include might be:
- maybe a location?
- if you want the ability to tag other users, maybe something to store a list of fellow users?
- any other post-related metadata you can think of

Reply:
- Photo